plugins/c8y_remote_access_plugin/src/config.rs

```diff
@@ -0,0 +1,40 @@
+use std::path::Path;
+use std::path::PathBuf;
+
+use miette::IntoDiagnostic;
+use tedge_config::C8yUrlSetting;
+use tedge_config::ConfigSettingAccessor;
+use tedge_config::TEdgeConfig;
+
+pub struct C8yUrl(pub String);
+
+impl C8yUrl {
```

Comment:
Not sure if this single parameter struct is very useful either as the user could always read this single value from `TEdgeConfig`. It made more sense in its earlier `PluginConfig` form when it had more parameters that had to be extracted from the `TEdgeConfig`.